ios-xcframework/Gutenberg/Gutenberg.h

```diff
@@ -0,0 +1,4 @@
+#import <Foundation/Foundation.h>
+
+FOUNDATION_EXPORT double GutenbergVersionNumber;
+FOUNDATION_EXPORT const unsigned char GutenbergVersionString[];
```

Comment:
I couldn't find where these values are populated from. Will this be part of a next PR?